diff --git PathOfBuildingBuild/src/Modules/Main.lua PathOfBuildingBuild/src/Modules/Main.lua
index 026fe6f9..7de83b43 100644
--- PathOfBuildingBuild/src/Modules/Main.lua
+++ PathOfBuildingBuild/src/Modules/Main.lua
@@ -44,11 +44,11 @@ function main:Init()
 	self.modes["LIST"] = LoadModule("Modules/BuildList")
 	self.modes["BUILD"] = LoadModule("Modules/Build")
 
-	if launch.devMode or (GetScriptPath() == GetRuntimePath() and not launch.installedMode) then
+	if false then
 		-- If running in dev mode or standalone mode, put user data in the script path
 		self.userPath = GetScriptPath().."/"
 	else
-		self.userPath = GetUserPath().."/Path of Building/"
+		self.userPath = GetUserPath( ) .. "/Documents/Path of Building/"
 		MakeDir(self.userPath)
 	end
 	self.defaultBuildPath = self.userPath.."Builds/"
@@ -191,7 +191,7 @@ function main:Init()
 	self.controls.versionLabel.label = function()
 		return "^8Version: "..launch.versionNumber..(launch.versionBranch == "dev" and " (Dev)" or launch.versionBranch == "beta" and " (Beta)" or "")
 	end
-	self.controls.devMode = new("LabelControl", {"BOTTOMLEFT",self.anchorMain,"BOTTOMLEFT"}, 0, -26, 0, 20, colorCodes.NEGATIVE.."Dev Mode")
+	self.controls.devMode = new("LabelControl", {"BOTTOMLEFT",self.anchorMain,"BOTTOMLEFT"}, 0, -26, 0, 20, colorCodes.NEGATIVE.."")
 	self.controls.devMode.shown = function()
 		return launch.devMode
 	end
@@ -206,7 +206,7 @@ function main:Init()
 	self.mainBarHeight = 58
 	self.toastMessages = { }
 
-	if launch.devMode and GetTime() >= 0 and GetTime() < 15000 then
+	if false and GetTime() >= 0 and GetTime() < 15000 then
 		t_insert(self.toastMessages, [[
 ^xFF7700Warning: ^7Developer Mode active!
 The program is currently running in developer
@@ -933,7 +933,7 @@ end
 
 function main:OpenUpdatePopup()
 	local changeList = { }
-	local changelogName = launch.devMode and "../changelog.txt" or "changelog.txt"
+	local changelogName = "changelog.txt"
 	local changelogFile = io.open(changelogName, "r")
 	if changelogFile then
 		changelogFile:close()
@@ -971,7 +971,7 @@ function main:OpenAboutPopup(helpSectionIndex)
 	local textSize, titleSize, popupWidth = 16, 24, 810
 	local changeList = { }
 	local changeVersionHeights = { }
-	local changelogName = launch.devMode and "../changelog.txt" or "changelog.txt"
+	local changelogName = "changelog.txt"
 	local changelogFile = io.open(changelogName, "r")
 	if changelogFile then
 		changelogFile:close()
@@ -992,7 +992,7 @@ function main:OpenAboutPopup(helpSectionIndex)
 	local helpSections = { }
 	local helpSectionHeights = { }
 	do
-		local helpName = launch.devMode and "../help.txt" or "help.txt"
+		local helpName = "help.txt"
 		local helpFile = io.open(helpName, "r")
 		if helpFile then
 			helpFile:close()
